src/components/toast/toast.component.tsx

```diff
@@ -176,9 +222,9 @@ export const Toast = React.forwardRef<HTMLDivElement, ToastProps>(
               onBlur: () => setTabIndex(undefined),
             })}
           >
-            {variant !== "notice" && (
-              <TypeIcon variant={variant}>
-                <Icon type={variant} />
+            {!isNotice && (
```

Comment:
**thought**: On second viewing I noticed that `ToastStyle` and `TypeIcon` are both dependent on `Message`'s internal styled components, even though we are actually overwriting their inherited styles with similar style rules.

This dependency with `Message` could lead to unnecessary complexity later on as further changes are made to `Toast` like we did in this PR.
 
Therefore it might be worth us in the future coming back to refactor `toast.style.ts`, so `ToastStyle` and `TypeIcon` are defined independently to `Message`'s internals.

Reply:
Yeah, agreed. I think when it was decided in 2019 to have `Toast` use `Message` under the hood, it was probably the correct decision but as the new Design System is evolving, we will need to uncouple these so a change in one doesn't cause headaches in the other.

Reply:
Yeah, agreed. I think when it was decided in 2019 to have `Toast` use `Message` under the hood, it was probably the correct decision but as the new Design System is evolving, we will need to uncouple these so a change in one doesn't cause headaches in the other.